lib/cocoapods/generator/copy_resources_script.rb

```diff
@@ -100,6 +100,7 @@ def script
 set -e
 
 mkdir -p "${CONFIGURATION_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}"
+mkdir -p "${PODS_ROOT}/resources-checksums"
```

Comment:
This should be created in the build directory as it as an intermediate build result.